pkg/server/crypto/sha256.go

```diff
@@ -0,0 +1,21 @@
+package crypto
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+	"strings"
+)
+
+const SHA256Prefix = "sha256~"
+
+func TrimSHA256Prefix(code string) (string, bool) {
+	if !strings.HasPrefix(code, SHA256Prefix) {
+		return code, false
+	}
+	return strings.TrimPrefix(code, SHA256Prefix), true
+}
+
+func SHA256Token(token string) string {
+	h := sha256.Sum256([]byte(token))
```

Comment:
nit: you don't really need `h`

Reply:
wdym?

you cannot write `<expression>[0:]` in go.